modules/optimization/test/tests/optimizationreporter/general_opt/point_loads/main_seperate_multiapps.i

```diff
@@ -0,0 +1,102 @@
+[Optimization]
+[]
+
+[OptimizationReporter]
+  type = GeneralOptimization
+  objective_name = misfit_norm
+  parameter_names = 'parameter_results'
+  num_values = '3'
+[]
+
```

Comment:
I think you should have two reporter blocks and move this one up next to the generalOptimization reporter

```suggestion
[Reporters]
  [main]
    # We need to have an OptimizationData on the main app to allow the
    # transferring of the correct information.
     type = OptimizationData
    !include measure_data.i
[]
```